pages/22.What's New/docs.en.md

```diff
@@ -26,7 +52,7 @@ See [Cron Jobs](./../setup/cron_jobs.html)
 
 ###Froala editor
 
-This release switches CKEditor in favor of Froala editor which has a more polished look and functionality. 
+This release switches CKEditor in favor of Froala editor which has a more polished look and functionality.
```

Comment:
We are not ditching Froala in Mautic 3.0 - Perhaps you mean the file manager? In that case please see the comments on the slack thread.

Reply:
Sorry, I completely misread and thought this was in the 3.x part! :blush: